layouts/partials/footer.html

```diff
@@ -27,6 +29,4 @@
       {{ partial "footer_links/footer-copyright.html" . }}
```

Comment:
The copyright section of the footer seems to be missing the design call out, was that removed from the final site?

Reply:
I'll add it ◡̈